ballerina-tests/np-tests/main.bal

```diff
@@ -20,9 +36,52 @@ type SportsPerson record {|
     string sport;
 |};
 
-function getPopularSportsPerson(
-        string nameSegment,
-        int decadeStart,
-        np:Prompt prompt = `Who is a popular sportsperson that was born in the decade starting
-            from ${decadeStart} with ${nameSegment} in their name?`)
-    returns SportsPerson|error? = @np:NaturalFunction external;
+function getPopularSportsPerson(string nameSegment, int decadeStart)
+      returns SportsPerson|error? => natural {
+    Who is a popular sportsperson that was born in the decade starting
+    from ${decadeStart} with ${nameSegment} in their name?
+};
+
+# Represents a place of interest.
+type PlaceOfInterest record {|
+    # Name of the place.
+    string name;
+    # City in which the place is located.
+    string city;
+    # Country in which the place is located.
+    string country;
+    # One-liner description of the place.
+    string description;
+|};
+
+function getPlacesOfInterest(string country, 
+                             string interest,
+                             int count = 3) returns PlaceOfInterest[]|error => natural {
+    Tell me about places in the specified country that could be a good destination 
+    to someone who has the specified interest.
+
+    Include only the number of places specified by the count parameter.
```

Comment:
We need to remove this `ballerina-tests` folder from the codecov using codecov configs
https://github.com/ballerina-platform/module-ballerina-data.csv/blob/41c2df6fc85f2562bb89957e38f8dec997d8b4d4/codecov.yml#L5